cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsUpdateService.java

```diff
@@ -43,20 +44,23 @@ public class AwsUpdateService {
     @Inject
     private AwsLaunchTemplateUpdateService awsLaunchTemplateUpdateService;
 
-    public List<CloudResourceStatus> update(AuthenticatedContext authenticatedContext, CloudStack stack, List<CloudResource> resources) {
+    public List<CloudResourceStatus> update(AuthenticatedContext authenticatedContext, CloudStack stack, List<CloudResource> resources, UpdateType type) {
```

Comment:
could you add some logs here so it would be easy to find out which path is taken?